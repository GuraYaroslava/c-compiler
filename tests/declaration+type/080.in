struct st1 { struct {}; } st;