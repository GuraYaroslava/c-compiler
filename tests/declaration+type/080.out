[Ln: 1, Col: 24] Error: exepcted an identifier
